public/index.php

```diff
@@ -86,6 +86,8 @@
 require __DIR__ . '/../src/app/vulnerabilities/rubyOnRailsSecretKeyDisclosure.php';
 require __DIR__ . '/../src/app/vulnerabilities/laravelTelescope.php';
 require __DIR__ . '/../src/app/vulnerabilities/404Check.php';
+require __DIR__ . '/../src/app/vulnerabilities/404with200statuscode.php';
+require __DIR__ . '/../src/app/vulnerabilities/404with200statuscodeWithReflectedPath.php';
 require __DIR__ . '/../src/app/vulnerabilities/text4shell.php';
```

Comment:
`2FABypass.php` should also be added in this file

Reply:
File names starting with digit is not very good idea, and please update the file name with camel casting

Reply:
`2FABypass.php` is an empty file